### ISSUE TITLE
2018-03-30 Public User Feedback Meeting - Node.js Tooling

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#45]
# 2018-03-30 Public User Feedback Meeting - Node.js Tooling

## Time

Friday, 2018-03-30 at 17:00 UTC / noon ET / 11am CT / 9am PT

## Links

* **Minutes Doc**: 
* **Previous Minutes Doc**: 

## Agenda

This is a public User Feedback meeting with [invited participants](https://github.com/nodejs/user-feedback/issues/38) focused on specific topics. Please feel free to propose discussion items for the agenda related to this topic below. Create a new issue to propose a new topic. Normally, @nodejs/user-feedback [team meetings](https://github.com/nodejs/user-feedback/labels/neuf-meeting) are extracted from [neuf-agenda](https://github.com/nodejs/user-feedback/labels/neuf-agenda) labelled issues and pull requests from the [user-feedback](https://github.com/nodejs/user-feedback) repo prior to the meeting.

* Introduction to the Node.js Tooling with @boneskull
* Topics (ref: #38):
  - Describe how your Tooling leverages Node.js.
  - Why do you use Node.js for Tooling?
  - What's working in the Node.js tooling ecosystem?
  - What isn't working in the Node.js tooling ecosystem?
  - What's new that could impact Node.js?
* Open feedback forum

## Invited
* User Feedback team members
  * Dan Shaw (@dshaw - User Feedback Champion, Mentorship, CommComm)
  * Mihai Ene-Pietrosanu (@mihaiep, User Feedback)
  * William Kapke (@williamkapke - CommComm, Individual Member Director)
  * Michael Dawson (@mhdawson - User Feedback, CommComm, TSC Chair)
  * Tierney Cyren (@bnb - User Feedback, CommComm Chair)
  * Tracy Hinds (@hackygolucky - CommComm)

* Node.js Tooling Representatives
  * @boneskull - Mocha
  * @TheLarkInn - webpack
  * @iarna - npm

## Public participation

The Node.js project live streams meetings so anyone can tune in. Everyone is invited to participate in the live chat. The stream will start playing at https://www.youtube.com/c/nodejs+foundation/live once we go live.

There's usually a short cat-herding time at the start of the meeting and occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and we'll be online shortly. ❤️

## Joining the meeting

Join link for invited members and participants:  https://zoom.us/j/204295944

There is a standing invitation for all @nodejs/TSC and @nodejs/community-committee members to attend in observer capacity.

**Comment by bnb** [comment]
@dshaw a few people were left out from https://github.com/nodejs/user-feedback/issues/37:

Also invited:
- @jdalton 
- @developit
- @bahmutov 
- @abernix
- @devsnek

**Comment by bnb** [comment]
Updated Zoom info as the previously linked meeting didn't exist.

**Comment by dshaw** [comment]
Updated Zoom link for those joining us: https://zoom.us/j/219327501

**Comment by developit** [comment]
@dshaw @bnb - I don't think I will be able to make it sadly, I'm stuck at the DMV.

**Comment by kentcdodds** [comment]
Feedback as requested by @bnb.

I love the meeting. Sadly due to time constraints we didn't all get a chance to address the questions. I think in the future it'd be great to make certain that the meeting is planned such that there is time to cover everything whether that means fewer questions, fewer attendees, shorter answers, or a combination.

**Comment by iarna** [comment]
Point of order: Making everyone a co-host means no one can raise hands using the Zoom interface, AFAICT.

**Comment by boneskull** [comment]
Need to compress that agenda.  think we tried to bite off more than we could chew.  would love to do this on a monthly cadence

**Comment by kentcdodds** [comment]
What's new that could impact Node.js?
- I'm hearing a lot about languages that compile down to native (ReasonML, Rust, etc.). I haven't dug into those much, but they could be alternatives for people and they seem to be much faster.
- Zeit CLI bundler thing called [pkg](https://github.com/zeit/pkg): "Single-Command Node.js binary compiler"
- Electron and similar tools, not really new, but I'd love more efforts around that for creating tooling that has a UI. This would be good for the users of node-based tools.

**Comment by boneskull** [comment]
Thanks everyone for the feedback.  This has been productive--yet there's also a lot more to talk about.

One of those future things might want to be "feature requests."  Some of us (e.g. @TheLarkinn) have very specific features in mind; others are more akin to absorbing the functionality of some important userland modules into core.

I would like to form a group in an "official" capacity, and would love to continue to discuss with all of the meeting attendees.  @dshaw let's work on that! :smile:

**Comment by devsnek** [comment]
i apologize for leaving early, my school's wifi died. (the rumor is that the fibre modem caught on fire so thats cool i guess :p)

**Comment by bnb** [comment]
Also, as per the discussion toward the end we wanted to capture the chat as there was some pretty interesting and important discussion to point to there.

I captured it in a Gist (there are two files, one formatted and one unformatted): https://gist.github.com/bnb/ff9d8e95876737f88f8874e36169df6b

**Comment by dshaw** [comment]
Thanks, @bnb.

@mihaiep, please note https://github.com/nodejs/user-feedback/issues/45#issuecomment-377595308 for minutes.

**Comment by mihaiep** [comment]
Yes. Thanks Tierney for the files. I have all the data, and will be available one minutes single doc for reviewing&check-in user repo.
